### ISSUE TITLE
Copy compiled command to clipboard

### ISSUE THREAD

**Issue by GnaspGames** [GnaspGames/Smelt#19]
Add an option (`--clipboard`) that will allow the final compiled command to be automatically placed in the users clipboard to place into the command-block in the world. 

This would only work when there is ONE final compiled command. If there are multiple outputs (because multiple files are processed, or because a !command has a setup command to run) then it wouldn't be possible to place them all into the clipboard and be usable. Possibly a future update could pause with each outputted command and allow the user to do one at a time?

**Comment by GnaspGames** [comment]
Decided to use the `--copy` switch as that seems more appropriate. I could allow a `--paste` switch in the future so that the input would be read from a clipboard and not a file. 

@M4GNV5 Thanks for your input on this, but I've used the 'copy-paste' module instead. That apparently handles cross platform clipboard systems. Although I've only tested on Windows so far. 

p.s. Been away on holiday so that's why there's not been much activity on this.
